src/main/java/frc/robot/triggers/DpadTrigger.java

```diff
@@ -0,0 +1,102 @@
+package frc.robot.triggers;
+
+import edu.wpi.first.wpilibj.XboxController;
+import edu.wpi.first.wpilibj2.command.button.Trigger;
+
+/**
+ * A trigger that responds to the Dpad of an xbox controller
+ */
+public class DpadTrigger extends Trigger {
+
+    /**
+     * Creates a Dpad Trigger
+     * 
+     * @param controller The controller to use
+     * @param angle      The angle of dpad button (0 is up)
+     */
+    public DpadTrigger(XboxController controller, int angle) {
```

Comment:
Can all these angles just be a nice enum or similar, instead of having tons of nearly identical functions?